tests/data/test_trade_converter_kraken.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime, timezone
+from pathlib import Path
+from shutil import copytree
+from unittest.mock import PropertyMock
+
+import pytest
+
+from freqtrade.data.converter.trade_converter_kraken import import_kraken_trades_from_csv
+from freqtrade.data.history.idatahandler import get_datahandler
+from freqtrade.exceptions import OperationalException
+from tests.conftest import EXMS, log_has, log_has_re, patch_exchange
+
+
+def test_import_kraken_trades_from_csv(testdatadir, tmpdir, caplog, default_conf_usdt, mocker):
+    with pytest.raises(OperationalException, match="This function is only for the kraken exchange"):
+        import_kraken_trades_from_csv(default_conf_usdt, 'feather')
+
+    default_conf_usdt['exchange']['name'] = 'kraken'
+
+    patch_exchange(mocker, id='kraken')
+    mocker.patch(f'{EXMS}.markets', PropertyMock(return_value={
+        'BCH/EUR': {'symbol': 'BCH/EUR', 'id': 'BCHEUR', 'altname': 'BCHEUR'},
+    }))
+    tmpdir1 = Path(tmpdir)
+    dstfile = tmpdir1 / 'BCH_EUR-trades.feather'
+    assert not dstfile.is_file()
+    default_conf_usdt['datadir'] = tmpdir1
+    # There's 2 files in this tree, containing a total of 2 days.
+    # tests/testdata/kraken/
+    # └── trades_csv
+    # ├── BCHEUR.csv       <-- 2023-01-01
+    # └── incremental_q2
+    #     └── BCHEUR.csv   <-- 2023-01-02
+
+    copytree(testdatadir / 'kraken/trades_csv', tmpdir1 / 'trades_csv')
+
+    import_kraken_trades_from_csv(default_conf_usdt, 'feather')
+    assert log_has("Found csv files for BCHEUR.", caplog)
+    assert log_has_re(r"BCH/EUR: 340 trades.* 2023-01-01.* 2023-01-02.*", caplog)
+
+    assert dstfile.is_file()
+
+    dh = get_datahandler(tmpdir1, 'feather')
+    trades = dh.trades_load('BCH_EUR')
+    assert len(trades) == 340
+
```

Comment:
Can you leave a comment explaining what this is for? Just curious :D

Reply:
i'm not sure what exactly you're asking ...

it's a test loading csv files from the test directory

* both csv files are loaded (one file is for the 1st, one for the 2nd)
* id is not filled (this would be a bug, the csv files don't contain 'id'